package.json

```diff
@@ -76,12 +76,16 @@
     ]
   },
   "eslintConfig": {
-    "extends": "wordpress",
+    "extends": "plugin:@wordpress/eslint-plugin/recommended",
```

Comment:
@mor10 this is the main update. I then ran ESLint and fixed all the errors surfaced when actually checking the current WP coding standards